diff --git a/android/src/main/java/expo/modules/kotlin/modules/Module.kt b/android/src/main/java/expo/modules/kotlin/modules/Module.kt
--- a/android/src/main/java/expo/modules/kotlin/modules/Module.kt
+++ b/android/src/main/java/expo/modules/kotlin/modules/Module.kt
@@ -6,7 +6,6 @@ import expo.modules.kotlin.AppContext
 import expo.modules.kotlin.runtime.Runtime
 import expo.modules.kotlin.convertToString
 import expo.modules.kotlin.providers.AppContextProvider
-import expo.modules.kotlin.tracing.trace
 import expo.modules.kotlin.types.Enumerable
 import expo.modules.kotlin.types.TypeConverterProvider
 import kotlinx.coroutines.CoroutineScope
@@ -67,11 +66,11 @@ abstract class Module : AppContextProvider {

 @Suppress("FunctionName")
-inline fun Module.ModuleDefinition(crossinline block: ModuleDefinitionBuilder.() -> Unit): ModuleDefinitionData {
-  return trace("${this.javaClass}.ModuleDefinition") { ModuleDefinitionBuilder(this).also(block).buildModule() }
+inline fun Module.ModuleDefinition(block: ModuleDefinitionBuilder.() -> Unit): ModuleDefinitionData {
+  return ModuleDefinitionBuilder(this).also(block).buildModule()
 }

 @Suppress("FunctionName")
-inline fun Module.ModuleConverters(crossinline block: ModuleConvertersBuilder.() -> Unit): TypeConverterProvider {
-  return trace("${this.javaClass}.TypeConverters") { ModuleConvertersBuilder().also(block).buildTypeConverterProvider() }
+inline fun Module.ModuleConverters(block: ModuleConvertersBuilder.() -> Unit): TypeConverterProvider {
+  return ModuleConvertersBuilder().also(block).buildTypeConverterProvider()
 }
diff --git a/android/src/main/java/expo/modules/kotlin/types/descriptors/typeDescriptorOf.kt b/android/src/main/java/expo/modules/kotlin/types/descriptors/typeDescriptorOf.kt
--- a/android/src/main/java/expo/modules/kotlin/types/descriptors/typeDescriptorOf.kt
+++ b/android/src/main/java/expo/modules/kotlin/types/descriptors/typeDescriptorOf.kt
@@ -32,7 +32,8 @@ internal fun PTypeDescriptor.toRawTypeDescriptor(): RawTypeDescriptor {
 @PublishedApi
 internal inline fun <reified T> ctTypeDescriptorOf(): TypeDescriptor {
   val typeDescriptor = pikaTypeDescriptorOf<T>().toRawTypeDescriptor()
-  val kTypeProvider = { typeOf<T>() }
+  val kType = typeOf<T>()
+  val kTypeProvider = { kType }

   return TypeDescriptor(
     typeDescriptor,
