diff --git a/ios/CryptoModule.swift b/ios/CryptoModule.swift
--- a/ios/CryptoModule.swift
+++ b/ios/CryptoModule.swift
@@ -15,13 +15,10 @@
 
     Function("digest", digest)
 
-    Function("randomUUID", randomUUID())
+    Function("randomUUID") { () -> String in
+      return UUID().uuidString.lowercased()
+    }
   }
-
-  @OptimizedFunction
-  private func randomUUID() -> String {
-    return UUID().uuidString.lowercased()
-  }
 }
 
 private func digestString(algorithm: DigestAlgorithm, str: String, options: DigestOptions) throws -> String {
